### DISCUSSION TITLE
How do I customize the style and text of internally-defined columns?

### DISCUSSION THREAD

**Discussion by nathan-fletcher-seagull-com** [KevinVandy/material-react-table#215] [Q&A] [answered]
I haven't found how to customize the style and text for internally-defined columns (such as the "Move" column after you enable row dragging). I have found props like muiTableBodyRowDragHandleProps that are great and useful, but I haven't been able to figure out how to change the header text from "move" to an empty string or an icon, or how to do something like decreasing the padding for just that column.

**Comment by KevinVandy** [comment, marked as answer]
You're looking for the [display column docs](https://www.material-react-table.com/docs/guides/display-columns)

You can change everything about the built-in columns, including their headers with the `displayColumnDefOptions` prop
